lib/ansible/plugins/callback/telegraf.py

```diff
@@ -0,0 +1,290 @@
+# (C) 2017-2018, VMware, Inc. All Rights Reserved.  
+# SPDX-License-Identifier: GPL-3.0
+# (C) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: telegraf
+    callback_type: metrics
+    requirements:
+      - callback_whitelist = telegraf[,others] in configuration
```

Comment:
This section is for requirements, not install instructions, those are generic for callbacks and/or can be addressed in description/notes or examples.

Reply:
moved